python/lib/eeg.py

```diff
@@ -301,6 +304,19 @@ def register_raw_data(self):
             files_to_archive = files_to_archive + (self.data_dir + fdt_file_path,)
         if event_file_path:
             files_to_archive = files_to_archive + (self.data_dir + event_file_path,)
+        if annotation_file_paths:
```

Comment:
if annotation are a type of derived data, shouldn't the handling of annotations be done under the section of `registere_derivatives_data` instead of being in the `register_raw_data` function?

Reply:
The problem here is that register_derivatives_data only processes derivatives eeg. It depends on how we decide to store annotations, but most likely annotations will be embedded in the source folder and target a raw eeg file, which will not fit the case covered by register_derivatives_data.